.circleci/config.yml

```diff
@@ -111,6 +111,27 @@ jobs:
       - run: npx typescript-json-schema --required --noExtraProps --out config-schema.json functions/src/config.ts Config
       - run: npx --package ajv-cli ajv validate -s config-schema.json -d config.json
 
+  prod-check-config:
+    environment:
+      FIREBASE_PROJECT: prod
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run: node_modules/.bin/firebase --project=$FIREBASE_PROJECT --token=$FIREBASE_TOKEN functions:config:get > config.json
+      - run: npm run config:check
+
+  prod-check-config-test:
```

Comment:
A generic test job for the tooling part would be better I think :) eg `test-tooling`